content/kapp/docs/latest/config.md

```diff
@@ -71,7 +71,8 @@ diffMaskRules:
 - `rebaseRules` (array) list of rebase rules
   - `path` (array) specifies location within a resource to rebase. Mutually exclusive with `paths`. Example: `[spec, clusterIP]`
   - `paths` (array of paths) specifies multiple locations within a resource to rebase. This is a convenience for specifying multiple rebase rules with only different paths. Mutually exclusive with `path`. Available in v0.27.0+.
-  - `sources` (array of strings) specifies source preference from where to copy value from. Allowed values: `new` or `existing`. Example: `[new, existing]`
+  - `type` (string) specifies strategy to get field values. Allowed values: `copy`
```

Comment:
There are two supported values for `type`: `copy` and `remove`

https://github.com/vmware-tanzu/carvel-kapp/blob/e62d79a86d884c40cb1ed49235bda7e116234834/pkg/kapp/config/config.go#L213

It would be helpful to give even a short description of how each value affects the rebase of the path(es).